federation-js/src/composition/validate/sdl/__tests__/matchingEnums.test.ts

```diff
@@ -106,7 +105,11 @@ describe('matchingEnums', () => {
           "code": "ENUM_MISMATCH",
           "locations": Array [
             Object {
-              "column": 11,
+              "column": 1,
```

Comment:
Noting that some of the diff here was actually expected in a previous commit when we went from passing `definitions[0]` to `definitions` the array. I almost made a comment on that commit since I was a little surprised there was no diff in tests, but figured I'd wait til later and here we are :)